source/plugin/commands/argumentparsing.rst

```diff
@@ -0,0 +1,176 @@
+================
+Argument Parsing
+================
+
+.. javadoc-import::
+    org.spongepowered.api.command.parameter.CommandContext
+    org.spongepowered.api.command.parameter.Parameter
+    org.spongepowered.api.command.Parameter.Parameter.Builder
```

Comment:
```suggestion
    org.spongepowered.api.command.parameter.Parameter.Builder
```

Reply:
Also the import doesn't seem to work.

Reply:
Interested, i dont get those build errors when building. I can see its wrong in the code, but wasnt notified so never made me look. Ill use your reference for the time being. But is there a specific command to do javadoc building?

Reply:
- https://github.com/SpongePowered/SpongeDocs/pull/959
- `cd tools/javadoc-checker && mvn`

Reply:
> I think I fixed all of them apart from the C import as thats a generic. Not exactly sure how to do generic parameter imports

IIRC you have to use the actual type instead of the generic name. e.g. `<T extends Actual> void foo(T bar)` -> `foo(Actual foo)`. Just make sure you can actually click on them. Theretically you can go to the jsdocs page and look for the correct value.

Reply:
Ah ok, then that should be fixed too